src/components/common.js

```diff
@@ -527,11 +527,15 @@ export const WarningTitle = ({ children }) => {
   ])
 }
 
-export const ClipboardButton = ({ text, onClick, ...props }) => {
+export const ClipboardButton = ({ text, onClick, style, ...props }) => {
   const [copied, setCopied] = useState(false)
   return h(Link, {
     ...props,
     tooltip: 'Copy to clipboard',
+    style: {
+      color: colors.accent(),
```

Comment:
@zarsky-broad only potentially controversial change since your last review: when I updated the URI viewer to re-use ClipboardButton I noticed that it was applying this color style. It felt like a reasonable idea to make this the default for all ClipboardButtons. Is this (plus the `...style` below) an appropriate way to do this?

Reply:
If you look at the definition of `Link`, it should be `colors.accent` by default, so you shouldn't even have to put this here, and then you can avoid destructuring style from props at all.